es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/ElasticSearchRestDAOV5.java

```diff
@@ -244,6 +246,12 @@ public void setup() throws Exception {
         } catch (IOException e) {
             logger.error("Failed to add {} mapping", TASK_DOC_TYPE);
         }
+
+        try {
+            Executors.newScheduledThreadPool(1).scheduleAtFixedRate(this::groomWorkflowsAndTasks, 0, 1, TimeUnit.DAYS);
```

Comment:
are we setting up two thread pool for grooming, does that explain why we are seeing this done multiple times in QA cloud?

Reply:
can you tell what part of the code is triggering 2 ?  It is not obvious to me.